test/Rings-conformance-tests.jl

```diff
@@ -731,6 +737,17 @@ function test_Poly_interface(Rx::AbstractAlgebra.PolyRing; reps = 30)
             @test is_monic(a) == isone(leading_coefficient(a))
          end
       end
+
+      @testset "Basic properties" begin
+         try
+            flag = is_irreducible(x)
+            @test flag || is_trivial(R)
+         catch e
+            if !(e isa NotImplementedError)
+               rethrow(e)
+            end
+         end
+      end
```

Comment:
This bit here is not yet done. The idea is to consider the test as passing if it throws a `NotImplementedError`. It's cumbersome this way (and not yet correct). Perhaps a `@test_if_implemented` macro could handle it nicer for use in other places. Or perhaps we investigate a completely different approach and e.g. add a trait `can_compute(R, is_irreducible)`, that may be more reliable...